extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -202,15 +200,6 @@ public function getDB( $type = DB_SLAVE ) {
 		$this->mDBhandler = wfGetDB( $type, array(), $this->mDBname );
 		$this->debug( "connecting to {$this->mDBname} via LoadBalancer" );
 
-		/**
-		 * if something goes wrong just fallback to $wgDBserver
-		 */
-		if( !$this->mDBhandler || !$this->mDBhandler->isOpen() ) {
-			error_log( "WikiFactoryLoader[{$this->mCityID}]: fallback to {$wgDBserver}" );
-			$this->mDBhandler = new DatabaseMysqli( $wgDBserver, $wgDBuser, $wgDBpassword, $this->mDBname );
-			$this->debug( "fallback to wgDBserver {$wgDBserver}" );
-		}
-
```

Comment:
Can we leave an `assert` here and make sure this edgecase does not happen?

Reply:
Good evening, Sir. :-)

If it happened, we would end up with a fatal error, as the initial value was a not-responding IP of a machine that has been already decommissioned. Although assert is not a bad idea.